src/test/java/webserver/data/JsonClientTest.java

```diff
@@ -0,0 +1,61 @@
+package webserver.data;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import webserver.data.JsonClient;
+import webserver.models.Todo;
+
+import static org.junit.Assert.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThan;
+
+@RunWith(MockitoJUnitRunner.class)
+@SuppressWarnings("unchecked")
+public class JsonClientTest {
+
+    //TODO setup a before to seed the json and an after to clear the json file. 
```

Comment:
Sounds like a good idea: make sure your tests all start with a known, clean slate.
You can test this empirically by altering the number / sequence of tests.  If you start getting different results based upon which order they run in, there's some sort of shared state (the files in the file system) that needs to be controlled within the test.

While I often suggest managing technical debt as a cost-benefit analysis of what issues you're currently experiencing, my experience is that this is often an area of technical debt that goes unnoticed and unmanaged, leads to people testing less instead of more, and generally leads to a lot of confusion in a project.

For here, it's easy enough to write a `@Before` method that deletes the file, creates a blank file, or copies a file with known contents -- whatever you need to know where you're starting, when your test code runs.  There's probably also a Junit4 `@Rule` for this, although I don't have much experience with those.  This is a case where I think it's helpful to roll your own a couple times, so that it's clear what the Rule API in JUnit is automating for you, if you use it later.